### PR TITLE
Build todo app

### PR DESCRIPTION
**Pull request by aimalamiri**
## Build the todo app
1. Remove all files from the src directory
2. Create new required components
3. Create the add task facility
4. Create the edit task facility
5. Create the delete task facility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,44 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Math Magician
```
Comment (marurevi) [inline README.md]: - [x] Remember to actualize the title of your readme 😉

**Review by marurevi — APPROVED** [review]
## Code Review
- ✅ Linters 
- ✅ Professional Gitflow
- ✅ Meaningful Commits

 You have done an amazing job on this project. I have some optional suggestions to improve it and make it look cleaner and dry.

- Fix the name of the project in the Readme file.
- Erase those files that you don't use like: src/logo.svg, src/reportWebVitals.js, src/setupTests.js or src/setupTests.js...

File: README.md (hunk 1 above)
Comment (aimalamiri) [reply README.md]: Thanks, I solved this in my last commit.

Comment (aimalamiri) [reply README.md]: The files src/log.svg src/reportWebVitals.js and others are already removed in this branch, I think you checked the dev, or main branch.

**Pull request merged**